<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CPDOLL" failures="0" tests="4" name="week1.ArrayIndicesMatchingTargetSum" time="0.013" errors="1" timestamp="2023-02-05T18:35:18 IST" skipped="0">
  <testcase classname="week1.ArrayIndicesMatchingTargetSum" name="example2" time="0.002"/>
  <system-out/>
  <testcase classname="week1.ArrayIndicesMatchingTargetSum" name="example4" time="0.002">
    <error message="No Matching indices" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: No Matching indices
at week1.ArrayIndicesMatchingTargetSum.getIndices(ArrayIndicesMatchingTargetSum.java:80)
at week1.ArrayIndicesMatchingTargetSum.example4(ArrayIndicesMatchingTargetSum.java:64)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- example4 -->
  <system-out/>
  <testcase classname="week1.ArrayIndicesMatchingTargetSum" name="example1" time="0.008"/>
  <system-out/>
  <testcase classname="week1.ArrayIndicesMatchingTargetSum" name="example3" time="0.001"/>
  <system-out/>
</testsuite> <!-- week1.ArrayIndicesMatchingTargetSum -->
